src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +24,33 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+
+    private static final int SIZE = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+
+        StringBuilder lines = new StringBuilder();
+        try (BufferedReader br = new BufferedReader(new FileReader(fileName))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                lines.append(line);
+            }
+        }
+        String text = lines.toString().toLowerCase()
+                .replaceAll("(\\d|\\W)", "");
```

Comment:
Try to do this using check from Character class (see the checklist). Also, you can form your set chars already in the try block